match/spec/storage/git_storage_spec.rb

```diff
@@ -12,286 +23,226 @@
     end
 
     describe "#download" do
-      it "clones the repo" do
-        path = Dir.mktmpdir # to have access to the actual path
-        expect(Dir).to receive(:mktmpdir).and_return(path)
-        git_url = "https://github.com/fastlane/fastlane/tree/master/certificates"
-        git_branch = "master"
-        shallow_clone = true
-
-        expected_commands = [
-          "git clone #{git_url.shellescape} #{path.shellescape} --depth 1 --no-single-branch",
-          "git --no-pager branch --list origin/#{git_branch} --no-color -r",
-          "git checkout --orphan #{git_branch}",
-          "git reset --hard"
-        ]
-        expected_commands.each do |command|
-          expect(FastlaneCore::CommandExecutor).to receive(:execute).once.with({
-            command: command,
-            print_all: nil,
-            print_command: nil
-          }).and_return("")
-        end
-
-        storage = Match::Storage::GitStorage.new(
-          git_url: git_url,
-          shallow_clone: shallow_clone
-        )
-        storage.download
+      describe "when no branch is specified" do
+        it("checkouts the master branch ") do
```

Comment:
Nit: Extra space, and it's generally a practice to avoid the parenthesis when declaring e.g. `describe`, `it`, etc.:

```suggestion
        it "checkouts the master branch" do
```

This should've been caught by rubocop but we're not enforcing this particular style yet 😅 could you change it throughout this file?